### DISCUSSION TITLE
Community Version Build to latest

### DISCUSSION THREAD

**Discussion by shekarpcs** [Netflix/conductor#3350] [General]
We have identified set-variable issue in 11.3 and we are not able to run 11.3 in production. When will the latest netflix code base will be merged to community version. 

It is stuck at 11.3 for long time. We are stuck at 10.7 as later version have set variable issues in do_while

**Comment by manan164** [comment]
Hi @shekarpcs , It is merged now, https://github.com/Netflix/conductor-community/blob/main/dependencies.gradle#L5
